internal/taskexec/manager.go

```diff
@@ -46,9 +46,10 @@ var (
 // Both cancelations and executions are started in detached context and run until completion.
 type Manager struct {
```

Comment:
If its logic is tied to local execution, should it be something like `LocalManager` as well?